lib/index.js

```diff
@@ -185,24 +188,21 @@
         // Descend through wrappers to the right content
         var j = map[0] + 1;
         while (j--) {
-          node = node.lastChild;
+          return node.lastChild.childNodes;
         }
       }
     } else {
       node.innerHTML = html;
-      node = node.lastChild;
+      return node.childNodes;
     }
-    return node;
-
-    // var holderNode = doc.createElement("div");
-    // holderNode.innerHTML = html;
-    // return holderNode.children[0];
   }
 
   function normalizeHTMLString(domString, attributesToConsider, attributesToExclude, stylesToConsider, classNamesToConsider) {
-    var dom = convertStringToDOM(domString);
-    var normalized = normalizeHTML(dom, attributesToConsider, attributesToExclude, stylesToConsider, classNamesToConsider);
-    return normalized;
+    var domNodes = convertStringToDOM(domString);
+
+    return Array.from(domNodes).map(function (domNode) {
```

Comment:
I wonder if this should be translated to something else by babel since it is ES6?!